modules/pricer/src/test/java/com/opengamma/strata/pricer/impl/swap/DiscountingRatePaymentPeriodPricerTest.java

```diff
@@ -797,6 +799,52 @@ public void test_explainPresentValue_single_gearingSpread() {
   }
 
   //-------------------------------------------------------------------------
+  public void test_currencyExposure_fx() {
```

Comment:
Same comment as for DiscountingFxResetNotionalExchangePricerTest:
Test currency exposure by finite difference: compare change of value of the product with change of value of the MultiCurrencyAmount currency exposure when the underlying exchange rate in FX_MATRIX is changed by a small amount.
The test should be done for a Payment which as not fixed and one which has fixed (i.e. the valuation date is between fixing and payment).